fwbot/tests/test_webhook.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from odoo.tests.common import HttpCase
+
+from .common import ForwardBotCase
+
+
+class TestWebhook(HttpCase, ForwardBotCase):
+    test_tags = {"standard", "post_install"}
```

Comment:
Use the standard way

Reply:
I did this temporarily because the decorator does not exist on Odoo 11, and to speed up development I am running unit tests on Odoo 16, as you can't choose to only run certain tests on Odoo 11, but must run the whole suit AFAIK.

Reply:
OK, but don't forget to remove it. I'll leave this thread opened until that.